__tests__/salesforce_uploader.js

```diff
@@ -17,15 +17,15 @@ jest.mock('../src/lib/storage', () => {
     copyObject: jest.fn(() => Promise.resolve('ok')),
     getObject: (path) => {
       if (validPaths.includes(path)) {
-        return Promise.resolve({Body: 'csv would be here', LastModified: new Date('12/30/2016')})
+        return Promise.resolve({ Body: 'csv would be here', LastModified: new Date('12/30/2016') })
       } else {
-        return Promise.reject({code: 'NoSuchKey'})// eslint-disable-line prefer-promise-reject-errors
+        return Promise.reject({ code: 'NoSuchKey' })// eslint-disable-line prefer-promise-reject-errors
```

Comment:
this could become:

`return Promise.reject(new Error('NoSuchKey'))`

I'll leave another comment in the implementation